pro/src/components/VenueForm/Activity/__specs__/Activity.spec.tsx

```diff
@@ -82,4 +82,26 @@ describe('components | Activity', () => {
       screen.getByText('Veuillez sélectionner un type de lieu')
     ).toBeInTheDocument()
   })
+
+  it('Should display less fields and infos when venue is virtual', async () => {
```

Comment:
~il manque les `expect(queryBy ...).not.toBeInTheDocument()`~

edit: Je viens de voir que la presence des champs est tésté plus haut dans VenueForm.